src/components/Heading/Heading.tsx

```diff
@@ -59,6 +59,10 @@ const getComputedAs = (size: HeadingSize) => {
 
 /**
  * `import {Heading} from "@chanzuckerberg/eds";`
+ *
+ * A component for styling heading text (`<h1>`-`<h6>`).
+ *
+ * Be careful to pass the correct heading element via the `as` prop to avoid skipping heading levels because [heading levels increasing by only one level at a time is important for screen reader users.](https://www.w3.org/WAI/tutorials/page-structure/headings/)
  */
 
```

Comment:
nit: I wonder why this one has a space between the comment block and the export... would this break the intellisense in VSCode?

Reply:
Do you mean the popover with the preview of the comment content? If so, that seems fine. I guess they've outsmarted empty lines between the components and their comments. (I vaguely remember an issue where the documentation wasn't appearing in storybook because of a space like this.)

<img width="732" alt="" src="https://user-images.githubusercontent.com/7761701/200424415-20f92274-fa77-46f5-90c8-401a8fd01ead.png">

Reply:
That being said, I can still remove the space.

Reply:
ha, yeah i remember it getting confused if the block wasn't right up against the code. Good to know it doesn't hurt nowadays